### DISCUSSION TITLE
Add a disable button for automation having non-existing entities.

### DISCUSSION THREAD

**Discussion by malosaa** [frenck/spook#430] [Ideas]
SPOOK is my homie now, seriously he is amazing to use in HA love it!!

would be crazy to add a disable automation button in this case:

![image](https://github.com/frenck/spook/assets/13116501/ab716f34-646a-469c-be9b-6c58b3f9d6ee)

U see that it asks me this :
```
To fix this error, [edit the automation](http://) and remove the use of these non-existing entities.
```
Also would be better that it ignore the disabled events
as for example i have 60+ issue's with automations but all of them are disabled automations and i did forget about them to even finish them haha.

Or we can even take it further:
I want it to be gone for ever.
Would be awesome to add a delete button (with a warning), so it actually removes the non functional part and disables the event.
But don't want SPOOK to turn into a demon :D

I like SPOOK!

**Comment by EarMaster** [comment]
I agree with you that disabled automations should not trigger warnings - or at least it should be an option.
